graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -460,7 +496,7 @@ def _split_classes_by_kind(self, class_name_to_definition):
         self._edge_class_names = frozenset(self._edge_class_names)
         self._non_graph_class_names = frozenset(self._non_graph_class_names)
 
-    def _set_up_schema_elements_of_kind(self, class_name_to_definition, kind, class_names):
+    def _set_up_schema_elements_of_kind(self, class_name_to_definition, kind_cls, class_names):
```

Comment:
I am not satisfied with kind_cls, but I will heavily refactor  _set_up_schema_elements_of_kind entirely in a future commit anyways. It is too large of a function and a bit hard to understand. I might also split it up into _set_up_vertex_elements, _set_up_edge_elements and _set_up_graph_elements.